timelock-server/src/integTest/java/com/palantir/atlasdb/timelock/LockWatchIntegrationTestUtilities.java

```diff
@@ -45,7 +47,8 @@
 
 public final class LockWatchIntegrationTestUtilities {
     private static final String TEST_PACKAGE = "package";
-    private static final String TABLE = "table";
+    static final String TABLE = "table";
+    static final String TABLE_2 = "rowLevelTable";
```

Comment:
This used to be called `"table2"`, but that lead to unexpected behaviour, because the entire-table watch for `table` actually _subsumes_ `table2`.